Sources/RichTextKit/Attributes/RichTextAttribute+Link.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  RichTextAttribute+Link.swift
+//
+//
+//  Created by Dominik Bucher on 01.12.2023.
+//
+
+import Foundation
+
+extension RichTextAttribute {
+    /**
+     This attribute ensures that we set the link and its color is always the one we choose.
+     Together with this, please set `linkTextAttributes = [:]` on UITextView instance. Otherwise
+     the tintColor will always override the foregroundColor. Or Follow RichTextView Configuration
+     */
+    static let richTextLink: NSAttributedString.Key = .init("richTextLink")
+}
+
+public struct CustomLinkAttributes {
+    init(link: String, color: ColorRepresentable) {
```

Comment:
Can we call this `RichTextLinkAttribute`? 

Also, what are the properties? Is `link` the `url`? If so, by renaming the type as suggested, we could use explicit naming and call this `url` or `urlString`.